baseplate/integration/pyramid.py

```diff
@@ -86,18 +86,38 @@ class ServerSpanInitialized(BaseplateEvent):
     pass
 
 
+class HeaderTrustHandlerInterface(object):
```

Comment:
just `HeaderTrustHandler` would be fine and would be more consistent with other interface base classes in baseplate.